protocols/v2/noise-sv2/src/aed_cipher.rs

```diff
@@ -1,16 +1,60 @@
+//! # AEAD Cipher Trait
```

Comment:
I get that the only living entity on the module is the `AeadCipher` trait, but this header should be talking about the module as whole.

```suggestion
//! # AEAD Cipher
```

Reply:
I'm not entirely sure about this, as the module only contains the `AeadCipher` trait and nothing else. I don't want readers to assume it includes anything beyond the `AeadCipher` trait. However, I'm also okay with the suggested change.